frontend/src/components/BrandImage.tsx

```diff
@@ -17,6 +17,10 @@ const BrandImage: React.FC<BrandImageProps> = ({ src, className, ...props }) =>
     'odh-card__header-fallback-img': !isSrcValid,
   });
 
+  React.useEffect(() => {
+    setIsSrcValid(true);
+  }, [src]);
+
```

Comment:
hmmmm: 
is there a need to check that `src` isn't empty?

future:
be nice to figure a unit test on this component, reading it as is... this fires every time yeah

Reply:
this will only fire when `src` changes. If `src` is empty, it will cause the onError call to set `srcValid` to false.
I suppose I can set it to false here if it is empty.